recipes-devtools/armnn-swig/armnn-swig.inc

```diff
@@ -66,3 +66,5 @@ def swiglib_relpath(d):
 do_install:append_class-native() {
     create_wrapper ${D}${bindir}/swig SWIG_LIB='`dirname $''realpath`'/${@swiglib_relpath(d)}
 }
+
+FILES:${PN} += "${datadir}/swig/*/*"
```

Comment:
This should be done on `-dev` package. If the `PN` is empty also add `ALLOW_EMPTY` for it.